### DISCUSSION TITLE
SidecarSet当业务容器挂掉的时候，会发TERM给sidecar容器吗？

### DISCUSSION THREAD

**Discussion by zxhdaniel** [openkruise/kruise#1105] [Q&A]
在测试的时候，发现业务容器挂掉而且restartPolicy=Never的时候，Pod的状态会先变成Error状态，过一会sidecar容器收到TERM信号，sidecar容器退出后，Pod状态为Completed。
这个TERM是SidecarSet的控制器发的吗？

**Comment by FillZpp** [comment]
不是，这种情况目前 Kruise 不会处理 sidecar 容器

**Reply by zxhdaniel** [reply]
那就奇怪了，那是谁把这个状态变成Completed的🤔？
